site/tutorials/keptn-litmus-018.md

```diff
@@ -0,0 +1,378 @@
+
+## Create a service
+Duration: 2:00
+
+After creating the project, services can be created for our project.
+For this purpose we need the helm charts as a tar.gz archive. To archive it use following command:
+
+<!-- command -->
+```
+cd helloservice && tar cfvz ./helm.tgz ./helm && cd ..
+```
+
+1. Create the **helloservice** service using the [keptn create service](https://keptn.sh/docs/0.10.x/reference/cli/commands/keptn_create_service/) and [keptn add-resource](https://keptn.sh/docs/0.10.x/reference/cli/commands/keptn_add-resource/)commands:
+
+    <!-- command -->
+    ```
+    keptn create service helloservice --project=litmus
+    keptn add-resource --project=litmus --service=helloservice --all-stages --resource=./helloservice/helm.tgz --resourceUri=helm/helloservice.tgz
+    ```
+
+1. After creating the service, tests need to be added as basis for quality gates. We are using JMeter tests, as the JMeter service comes "batteries included" with our Keptn installation. Although this could be changed to other testing tools, we are going with JMeter in this tutorial. Let's add some JMeter tests as well as a configuration file to Keptn.
+
+    <!-- command -->
+    ```
+    keptn add-resource --project=litmus --stage=chaos --service=helloservice --resource=./jmeter/load.jmx --resourceUri=jmeter/load.jmx
+    keptn add-resource --project=litmus --stage=chaos --service=helloservice --resource=./jmeter/jmeter.conf.yaml --resourceUri=jmeter/jmeter.conf.yaml
+    ```
+
+Now each time Keptn triggers the test execution, the JMeter service will pick up both files and execute the tests.
+
+## Configure Quality Gate
+Duration: 2:00
+
+We have not yet added our quality gate, i.e., the evaluation of several SLOs done by Keptn. Let's do this now!
+
+
+1. First, we are going to add an SLI file that holds all service-level indicators we want to evaluate along with their PromQL expressions. Learn more about the concept of [Service-Level Indicators in the Keptn docs](https://keptn.sh/docs/concepts/quality_gates/#what-is-a-service-level-indicator-sli).
+
+    ```
+    keptn add-resource --project=litmus --stage=chaos --service=helloservice --resource=./prometheus/sli.yaml --resourceUri=prometheus/sli.yaml
+    ```
+
+1. Now that we have added our SLIs, let us add the quality gate in terms of an `slo.yaml` which adds objectives for our metrics that have to be satisfied. earn more about the concept of [Service-Level Objectives in the Keptn docs](https://keptn.sh/docs/concepts/quality_gates/#what-is-a-service-level-objective-slo).
```

Comment:
The reference page for slo.yaml is https://keptn.sh/docs/0.18.x/reference/files/slo/